WordPress/Classes/Networking/PostServiceRemoteXMLRPC.m

```diff
@@ -164,6 +169,54 @@ - (void)deletePost:(RemotePost *)post
     }
 }
 
+- (void)trashPost:(RemotePost *)post
+          forBlog:(Blog *)blog
+          success:(void (^)(RemotePost *))success
+          failure:(void (^)(NSError *))failure
+{
+    NSParameterAssert([post.postID longLongValue] > 0);
+    NSNumber *postID = post.postID;
+    if ([postID longLongValue] > 0) {
+        NSArray *parameters = [blog getXMLRPCArgsWithExtra:postID];
+
+        WPXMLRPCRequest *deletePostRequest = [self.api XMLRPCRequestWithMethod:@"wp.deletePost" parameters:parameters];
```

Comment:
I disagree.  Whenever we have the second parameter in a call starting after the 80 character mark pressing enter once is really trivial and the gain in readability is worth it.

Take for example [this line](https://github.com/wordpress-mobile/WordPress-iOS/blob/issue/3506-post-list-part-3/WordPress/Classes/Networking/AccountServiceRemote.h#L9).